### PR TITLE
[webpack] Fix support for webpack and add example app

### PR DESCRIPTION
**Pull request by brijeshb42**
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/pigment-css/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: packages/pigment-css-unplugin/src/index.ts [hunk 1]
```diff
@@ -326,14 +324,20 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
...
-        const cssFilename = `${slug}.pigment.css`;
-        const cssId = `./${cssFilename}`;
-        cssFileLookup.set(cssId, cssFilename);
-        cssLookup.set(cssFilename, cssText);
+        const cssFilename = path
+          .normalize(`${id.replace(/\.[jt]sx?$/, '')}-${slug}.pigment.css`)
+          .replace(/\\/g, path.posix.sep);
+
+        const cssRelativePath = path.relative(rootPath, cssFilename).replace(/\\/g, path.posix.sep);
+        const cssId = `\0${cssRelativePath}`;
```
Comment (siriwatknp) [inline packages/pigment-css-unplugin/src/index.ts:334]: Why the `\0`? a comment would help a lot.

**Review by siriwatknp — APPROVED** [review]
👍 Overall, looks good. I have one minor request for the code comment.

We should look into the tests for theses plugins to verify that they don't break existing behavior.

File: packages/pigment-css-unplugin/src/index.ts (hunk 1 above)
Comment (brijeshb42) [reply packages/pigment-css-unplugin/src/index.ts:336]: Added

**Review by oliviertassinari — COMMENTED** [review]
👍 Great to have a working demo with webpack

**Pull request merged**